include/sys/file.h

```diff
@@ -25,14 +25,21 @@ enum { otDir = 0, otFile, otDev, otSymlink, otUnknown };
 enum { atMode = 0, atUid, atGid, atSize, atBlocks, atIOBlock, atType, atPort, atPollStatus, atEventMask, atCTime, atMTime, atATime, atLinks, atDev };
 
 
-enum { mtMount = 0xf50, mtUmount, mtSync, mtStat };
+enum { mtMount = 0xf50, mtUmount, mtSync, mtStat, mtMountPoint };
 
 
 typedef struct {
-	long id;
-	unsigned mode;
+	oid_t dev;
+	oid_t mnt;
+	unsigned long mode;
 	char fstype[16];
-} mount_msg_t;
+} __attribute__((packed)) mount_i_msg_t;
+
+
+typedef struct {
+	int err;
+	oid_t oid;
+} __attribute__((packed)) mount_o_msg_t;
```

Comment:
What do you think? One pascal less. Just a thought 
```suggestion
} __attribute__((packed)) mount_imsg_t;


typedef struct {
	int err;
	oid_t oid;
} __attribute__((packed)) mount_omsg_t;
```

Reply:
I tried to stick to the convention already used in our codebase, e.g. here: [link](https://github.com/phoenix-rtos/phoenix-rtos-devices/blob/9e1c8296894d8f33e7f8f67758cac2353c1b9a51/storage/imx6ull-flash/imx6ull-flashsrv.h#L74-L80)
I don't mind either approach.

Reply:
Ack